CorsixTH/Lua/dialogs/fullscreen/graphs.lua

```diff
@@ -41,6 +41,7 @@ local colours = {
 
 function UIGraphs:UIGraphs(ui)
   self:UIFullscreen(ui)
+  self.app = ui.app
```

Comment:
Maybe self.app = TheAp or else pass in app.

Reply:
To clarify, `TheApp` is a global constant pointing at `app`.

In other words, there is no need to have `self.app` anywhere in the program, as it's always available ass `TheApp`.